project/ClasspathSupport.scala

```diff
@@ -0,0 +1,6 @@
+import sbt._
+import sbt.Keys._
+
+object ClasspathSupport {
+  val LibraryClasspath = Configuration.of("ClasspathSupport.LibraryClasspath", "ClasspathSupport.LibraryClasspath")
```

Comment:
Any reason that this is not a top-level `val` in `build.sbt`?

Reply:
I copy pasted this from scala 2 where I did it this way for the tasty-tests, I can move it to build.sbt